packages/cli/README.md

```diff
@@ -34,6 +34,17 @@ glaze COMMAND
 - [`glaze model:inspect NAME`](#glaze-modelinspect-name)
 - [`glaze model:list`](#glaze-modellist)
 - [`glaze model:publish NAME [OUTPUT]`](#glaze-modelpublish-name-output)
+- [`glaze stream:pin:add STREAMID`](#glaze-streampinadd-streamid)
+- [`glaze stream:pin:rm STREAMID`](#glaze-streampinrm-streamid)
+- [`glaze stream:pin:ls STREAMID`](#glaze-streampinls-streamid)
+- [`glaze stream:schema:create CONTENT`](#glaze-streamschemacreate-content)
+- [`glaze stream:schema:update STREAMID CONTENT`](#glaze-streamschemaupdate-streamid-content)
+- [`glaze stream:commits STREAMID`](#glaze-streamcommits-streamid)
+- [`glaze stream:create STREAMID`](#glaze-streamcreate-streamid)
```

Comment:
That doesn't seem right, we should create the stream with contents and optional metadata arguments, no?